specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -4518,6 +4518,10 @@
       "properties": {
         "@odata.type": {
           "type": "string"
+        },
+		"name": {
+          "type": "string",
+          "description": "The name of the skill which uniquely identifies the skill within the skillset. Skill names must be unique per skillset."
```

Comment:
Please explain the fact that unique names are automatically assigned to skills if this property isn't set.